runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -138,9 +190,9 @@ public final class HttpClientFactory implements HttpStreamFactory
                          .wrap(new UnsafeBuffer(new byte[8]), 0, 8)
                          .build();
     private static final Map<String8FW, String16FW> EMPTY_OVERRIDES = emptyMap();
-
     private final BeginFW beginRO = new BeginFW();
     private final DataFW dataRO = new DataFW();
+    private final AtomicBuffer payloadRO = new UnsafeBuffer(0, 0);
```

Comment:
Can this be simplified to `DirectBuffer` type instead of `AtomicBuffer` type field?